### ISSUE TITLE
A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader: Levels are announced for normal text that is present on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9225]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#Keyboard;#MAS1.3.1; #MAS4.1.2;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#NVDA;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen Reader: NVDA.
MAS reference: MAS1.3.1, MAS4.1.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Navigate to File > New Query
3. Connect to a Server in the Query Editor and choose the credential, and optionally the database to connect to the server.
4. Type a SQL query statement.
5. Click Run.
6. Navigate to Messages section.
7. Check if all announcements are made properly by screen reader.

**Actual:**
Issue 1:
When we navigate to the data using arrow keys, after reading the text 'level 1' is announced by NVDA after every line.

Issue 2:
Tree control type should not be assigned to the messages panel where the data is being displayed.

**Expected:**
Issue 1:
'Level 1' keyword should not be announced as there is no hierarchy present for any of the data on the screen.

Issue 2:
'Text' control type should be assigned to the panel present on the screen inside messages screen.

**User Impact:**
Screen reader users might get confused as there is no hierarchy present for any of the data yet levels are announced for the data present on the screen. 

**Attachment link reference:**
[9225_A11y_SQLAzureDataStudio_Query_Create&Run_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291434/9225_A11y_SQLAzureDataStudio_Query_Create.Run_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
seems we shouldn't be using heading here, i will email the accessibility team for guideline on when to use h tag

**Comment by Harika1702** [comment]
<img width="850" alt="image" src="https://user-images.githubusercontent.com/100270074/218955918-0566b56b-5544-4a45-b8cc-908f8c22db19.png">
Verified this issue now not repro'ing.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
